tox.ini

```diff
@@ -9,7 +9,7 @@ envlist = py{27,36,37,py,py3}, docs, flake8
 [testenv]
 passenv = DISPLAY PYTEST_ADDOPTS
 deps = pytest-localserver
-commands = pytest -v -r a {posargs}
+commands = pytest -W error::DeprecationWarning -v -r a {posargs}
```

Comment:
This will cause any test to Fail if they cause any DeprecationWarnings to be invoked. @davehunt